[2023-12-10 19:43:36,352] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-06-16T00:00:00+00:00 [queued]>
[2023-12-10 19:43:36,608] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-06-16T00:00:00+00:00 [queued]>
[2023-12-10 19:43:36,624] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:43:36,626] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:43:36,627] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:43:36,745] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-06-16 00:00:00+00:00
[2023-12-10 19:43:36,856] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-06-16T00:00:00+00:00', '--job-id', '961', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmp_e1drdwd', '--error-file', '/tmp/tmp7bhj9opr']
[2023-12-10 19:43:36,869] {standard_task_runner.py:80} INFO - Job 961: Subtask segunda_tarea
[2023-12-10 19:43:36,773] {standard_task_runner.py:52} INFO - Started process 3417 to run task
[2023-12-10 19:43:37,287] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-06-16T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:43:38,095] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-06-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-16T00:00:00+00:00
[2023-12-10 19:43:38,101] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:43:38,103] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-12-10 19:43:38,142] {subprocess.py:85} INFO - Output:
[2023-12-10 19:43:38,155] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-12-10 19:43:38,156] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:43:38,438] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230616T000000, start_date=20231210T194336, end_date=20231210T194338
[2023-12-10 19:43:38,703] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:43:39,913] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
